charts/values.yaml

```diff
@@ -6,12 +6,27 @@ replicaCount: 1
 # Termination Grace Period
 terminationGracePeriod: 120
 
+# Group related settings
+group:
+  # the group name of the current Marklogic Helm Deployment
+  name: Default
+
+# The name of the host to join. If not provided, the deployment is a bootstrap host.
+bootstrapHostName: ""
+
+
 # Marklogic image parameters
 image:
   repository: marklogicdb/marklogic-db
   tag: latest
   pullPolicy: IfNotPresent
 
+# Init container image parameters
+initContainerImage:
+  repository: curlimages/curl
+  tag: latest
```

Comment:
We should avoid using latest tag for any docker image and use a proper versioned tag instead.
Latest tag has to be used only for Dev.